### ISSUE TITLE
Changes browser shows changes when Project is not dirty

### ISSUE THREAD

**Issue by ericwinger** [GemTalk/Rowan#631]
I opened a Changes Browser on the Rowan project. The SHA was 6ba5da8d and no modifications had been made to the project. 

Surprisingly, there were class comment changes. Looks like something related to class comments has changed as the class comment for the classes was formerly empty. 

![image](https://user-images.githubusercontent.com/36212516/89312082-acd80f00-d62b-11ea-8529-84508ab468cd.png)

**Comment by ericwinger** [comment]
Testing the commit and pushing to GitHub, I do see that the unchanged class comments were committed even though I didn't make the changes. 
https://github.com/GemTalk/Rowan/commit/f39c145859acd092015325c2ec7537ab6d33ff13

**Comment by dalehenrich** [comment]
Interesting ... I think I know where the comment is coming from, but I'm surprised that I haven't seen this bug before ... what you are getting is the default comment from GemStone when you send the comment message to class that has no comment ... I've got a recent build and I don't see that happening for me, so you might have done something to trigger this issue - for example my changes are clean and the Rowan Services classes don't show up with dirty comments ... perhaps you've changed how the services code gets comments or something

**Comment by ericwinger** [comment]
Looks like Jadeite sends `rwComment` and `rwComment:` to get and set the comment. Is that still correct?

**Comment by dalehenrich** [comment]
Yeah, those weren't changed when I did some recent comment work ... the change I made was to turn empty comment strings into a nil ... so I will guess that you (or the code you are using to do this diffs) is sending comment instead of rwComment ... because comment is the message that fabricates the default comment .... but I don't think that is the whole story

**Comment by ericwinger** [comment]
After talking with Dale, I'm going to back out sha 'f39c1458' to  '6ba5da8d'.  Then I'll attempt to recreate this problem.

**Comment by ericwinger** [comment]
Have a potential smoking gun here. In the latest commit of the checked out branch (oscarV2.0), the superclass of `RowanService` was changed. That forces a recompile of each subclass which might touch class comment code.  It would also explain why it was only service classes which are affected. 

![image](https://user-images.githubusercontent.com/36212516/89352411-e24f1d80-d668-11ea-856f-6cbce3b3ff90.png)
